base/event.jl

```diff
@@ -341,7 +341,7 @@ end
 ## timer-based notifications
 
 """
-    Timer(delay, repeat=0)
+    Timer(delay; repeat = 0)
```

Comment:
I think the `repeat` argument to `Timer` is kind of confusingly named – it seems like it gives a repeat count, but instead gives the interval of repetition. Perhaps it should be called `interval` instead with the same behavior? I.e. if given the timer repeats with the given interval, if not given the timer does not repeat. We might in the future want to add a `repeat` keyword argument which controls how many _times_ a timer repeats. Of course, we might want to call such an argument `count` instead, but you see my point.

Reply:
I was confused by `repeat` at first too. I'll change it to `interval`.